src/controllers/verifyCommand.ts

```diff
@@ -1,8 +1,12 @@
+import { VERIFICATION_SITE } from "../constants/urls";
 import { env } from "../typeDefinitions/default.types";
 import { discordTextResponse } from "../utils/discordResponse";
+import { generateUniqueToken } from "../utils/generateUniqueToken";
 import { sendDiscordDm } from "../utils/sendDiscordDm";
 
 export async function verifyCommand(userId: number, env: env) {
-  await sendDiscordDm(userId, env);
+  const token = await generateUniqueToken();
+  const url = `${VERIFICATION_SITE}/?token=${token}`;
+  await sendDiscordDm(userId, env, url);
```

Comment:
Here we are passing a URL but in the sendDiscordDm util, the parameter is a token. Is this correct or am I missing something here?
![image](https://user-images.githubusercontent.com/78433013/218696852-379d82e6-d97a-414d-811e-4a1e5b6c5ff2.png)

Reply:
sorry i missed that